### PR TITLE
Hit the api

### PR DESCRIPTION
**Pull request by username-forGithub**
- Send and receive data from an API.
- Use API documentation.
- Understand and use JSON.
- Make JavaScript code asynchronous.

### CODE REVIEW

File: src/modules/get.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  getUsers().then((val) => {
+    const newArr = val.sort((a, b) => parseFloat(b.score) - parseFloat(a.score));
+    let str = '';
+    newArr.forEach((el) => {
+      str += `
+            <li><span class="name">${el.user}: </span><span class="score">${el.score}</span></li>
+            `;
+    });
+    document.querySelector('ul').innerHTML = str;
+  });
```
Comment (roshan-bajgain) [inline src/modules/get.js:26]: - [ ] When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change. Please implement it

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @username-forGithub, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
@username-forGithub,

This is an additional review as a result of the second opinion request.

I can see you have all the necessary implementation required for this milestone. 

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

Keep rocking 🚀!

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @username-forGithub,

Your project is complete!  You did well on the following: 💪

✅ You used Gitflow correctly
✅ No linter error
✅ README is ok
✅ Good descriptive PR
✅ App works fine as expected 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**